script/Ciel/Day 7/07_01_CIEL07_8.txt

```diff
@@ -152,39 +156,44 @@ Shiki, how much do you understand about your eyes?"
 [sha:975d92975b842583d780e3f0ffb12feb565fbc37]{
 -- Page 632, Offset 16331.
 -- まるで教師のように、俺の“眼”について講義を始めた。
-She continues on, almost like a teacher.
+It's as if she's really a teacher or something.
 }
 [sha:a12b6552bb887ccc80db82785f40845dc4637f52]{
 -- Page 633, Offset 16332.
 -- ……アルクェイドによると、俺の眼は死を視ているのではなく、死を解読しているようなものなんだとか。
-According to Arcueid, my eyes aren't %{i}seeing%{/i} death, as much as they're %{i}deciphering%{/i} it,
+According to Arcueid, my eyes aren't %{i}seeing%{/i} death so much as %{i}deciphering%{/i} it.
 }
 [sha:50c6641cbb94a49bab14aa6283f5a85a692314c9]{
 -- Page 633, Offset 16333.
 -- 『線』はあくまで死の露出であり、その線の基点である『点』こそが死そのもの。
-Ultimately, the lines are just death laid bare, while the origination of those lines――the points――are death itself.
+// Most uses of 露出 that I turned up seem to be exposure in the context of photography (rather than to bare something),
```

Comment:
Meanwhile the context I see this word in most is porn... sure explains my interpretation...